### PR TITLE
Fix crashs with FastPlayerList update

### PR DESCRIPTION
**Pull request by MrGenga**
### Description

It fix crash with error about FastPlayerListPacket

### Reason to modify

Fix crash :)
### Tests & Reviews

I have tested the code and it works. Client doesn't freezes I check it, transfer per 4 servers and it doesn't freeze.

### CODE REVIEW

**Comment by dktapps** [comment]
DETAILS. What errors were caused, and why do your changes solve the problem? Also please squash your commits.

**Comment by MrGenga** [comment]
@dktapps I will ad note to code lines, ok

File: src/synapse/Player.php [hunk 1]
```diff
@@ -278,8 +278,10 @@ public function dataPacket(DataPacket $packet, $needACK = false){
 			$pk->sendTo = $this->uuid;
 			$pk->type = $packet->type;
 			foreach($packet->entries as $entry){
-				array_pop($entry);
-				array_pop($entry);
+				if($packet->type !== PlayerListPacket::TYPE_REMOVE){
```
Comment (MrGenga) [inline src/synapse/Player.php]: There is error, because entry has 2 unwanted parametrs, but only in TYPE_ADD, I added check of type to fix it.

**Comment by PeratX** [comment]
He fixed different types of PlayerListPacket.

**Comment by PeratX** [comment]
+1

**Comment by dktapps** [comment]
+1 and merge.

**Pull request merged**